src/Warning.less

```diff
@@ -0,0 +1,13 @@
+@import (less) "./base.less";
+
+.warning {
+    .close {
+        position: absolute;
+        right: 0;
+        top: 0;
+        margin: 5px 10px;
+        svg {
+            fill: @pacific;
```

Comment:
Blue and not black? Interesting

Reply:
I confirmed with Matt. he said interactive color #0072CE since its a clickable link